config/crd/bases/test.openstack.org_tobikoes.yaml

```diff
@@ -73,6 +77,8 @@ spec:
                 default: ""
                 description: Tobiko version
                 type: string
+            required:
```

Comment:
storage-class should not be required.

ATM the only requirement is being writable AFAIK.

Reply:
StorageClass is required when there is no default storage class present in the environment. Or am I missing something?